src/components/TeamsPricingCard.astro

```diff
@@ -0,0 +1,76 @@
+---
+import Tooltip from "./Tooltip.astro";
+
+export interface Props {
+  title: string;
+  description: string;
+  price: string;
+  price_details: string;
+  payment_details: string;
+  features: string[];
+  feature_tooltip: string[];
+}
+
+const {
+  title,
+  description,
+  price,
+  price_details,
+  payment_details,
+  features,
+  feature_tooltip,
+} = Astro.props;
+---
+
+<div
+  class="bg-gradient-to-br from-blue-600 to-brand-pink w-full sm:w-[360px] sm:h-[572px] rounded-[32px] p-[4px]"
+>
+  <div class="h-full w-full rounded-[28px] bg-gray-800 px-8 py-12">
+    <div class="flex flex-col justify-between h-full">
+      <div class="space-y-6 sm:space-y-8">
+        <div>
+          <h3 class="text-xl sm:text-2xl font-semibold text-yellow-600">
+            {title}
+          </h3>
+          <p class="text-lg font-[100] font-['Inter'] text-gray-200 sm:h-16">
```

Comment:
Let's use tailwind built-ins, instead of inlined values, whenever necessary. Also, Inter is already the default page font :)

```suggestion
          <p class="text-lg font-thin text-gray-200 sm:h-16">
```